runtime/parachains/src/disputes.rs

```diff
@@ -186,12 +186,12 @@ impl<T: Config> DisputesHandler<T::BlockNumber> for pallet::Pallet<T> {
 		pallet::Pallet::<T>::provide_multi_dispute_data(statement_sets)
 	}
 
-	fn note_included(
+	fn process_included(
 		session: SessionIndex,
 		candidate_hash: CandidateHash,
-		included_in: T::BlockNumber,
+		revert_to: T::BlockNumber,
```

Comment:
it's not exactly revert_to, though, because I think we revert to `included_in - 1`

Reply:
How about we rename it to `revert_tail`, since it's the tail of the chain that gets removed in reversion

Reply:
not an expert thus can't give good terminology advice, but seems like `included_in` was a beteter name, when there's a clear document saying `we revert to included_in - 1 when ...`.